docs/sources/datasources/tempo/configure-tempo-data-source.md

```diff
@@ -60,62 +60,86 @@ refs:
+
 To configure basic settings for the Tempo data source, complete the following steps:
 
-1.  Click **Connections** in the left-side menu.
-1.  Under Your connections, click **Data sources**.
+1.  Click **Connections** in the left-side menu and select **Data sources**.
 1.  Enter `Tempo` in the search bar.
 1.  Select **Tempo**.
+1.  On the **Settings** tab, complete the **Name**, **Connection**, and **Authentication** sections.
 
-1.  On the **Settings** tab, set the data source's basic configuration options:
-
-    | Name           | Description                                                              |
-    | -------------- | ------------------------------------------------------------------------ |
-    | **Name**       | Sets the name you use to refer to the data source in panels and queries. |
-    | **Default**    | Sets the data source that's pre-selected for new panels.                 |
-    | **URL**        | Sets the URL of the Tempo instance, such as `http://tempo`.              |
-    | **Basic Auth** | Enables authentication to the Tempo data source.                         |
-    | **User**       | Sets the user name for basic authentication.                             |
-    | **Password**   | Sets the password for basic authentication.                              |
+- Use the **Name** field to specify the name used for the data source in panels, queries, and Explore. Activate **Default** for the data source to be pre-selected for new panels.
+- Under **Connection**, enter the **URL** of the Tempo instance, for example, `https://example.com:4100`.
+- Complete the [**Authentication** section](#authentication).
 
-You can also configure settings specific to the Tempo data source.
+1. Optional: Configure other sections to add capabilities to your tracing data.
+1. Select **Save & test**.
 
-This video explains how to add data sources, including Loki, Tempo, and Mimir, to Grafana and Grafana Cloud. Tempo data source set up starts at 4:58 in the video.
+This video explains how to add data sources, including Loki, Tempo, and Mimir, to Grafana and Grafana Cloud.
+Tempo data source set up starts at 4:58 in the video.
 
 {{< youtube id="cqHO0oYW6Ic" start="298" >}}
 
+## Authentication
+
+Use this section to select an authentication method to access the data source.
+
+{{< admonition type="note" >}}
+Use Transport Layer Security (TLS) for an additional layer of security when working with Tempo.
+For additional information on setting up TLS encryption with Tempo, refer to [Configure TLS communication](https://grafana.com/docs/tempo/<TEMPO_VERSION>/configuration/network/tls/) and [Tempo configuration](https://grafana.com/docs/tempo/<TEMPO_VERSION>/configuration/).
+{{< /admonition >}}
+
+[//]: # 'Shared content for authentication section procedure in data sources'
+
+{{< docs/shared source="grafana" lookup="datasources/datasouce-authentication.md" leveloffset="+2" version="<GRAFANA_VERSION>" >}}
+
 ## Streaming
```

Comment:
@adrapereira I updated this section to reflect the changes in your PR. Would you verify that these updates are correct? 

Thank you.